frontend/src/scenes/ingestion/constants.ts

```diff
@@ -23,6 +23,7 @@ export const VERIFICATION = 'VERIFICATION'
 export const WEB = 'Web'
 export const MOBILE = 'Mobile'
 export const BACKEND = 'Backend'
+export const IMPORT = 'Import data via an app'
```

Comment:
Nit: I think `APP` might be a more consistent name to match the other options.

Reply:
yesss